omnibus/resources/agent/msi/source.wxs.erb

```diff
@@ -391,53 +383,6 @@
           <CreateFolder/>
         </Component>
       </Directory>
-
-      <Directory Id="TUFMETALOCATION" Name="repositories">
-         <Component Id="CleanupTUFMetaFolder" Guid="2031779f-2616-4eea-ac09-42ce47e07cc2">
-           <!-- We need to use TUFMETALOCATION variable here or RemoveFolderEx
-                will not remove on "install". -->
-           <RegistryValue Root="HKLM"
-                          Key="SOFTWARE\Datadog\Datadog Agent"
-                          Name="TufMetaPath"
```

Comment:
So I just verified this `TufMetaPath` key in the registry corresponds to the `C:\ProgramData\Datadog\repositories\` which is currently removed by the `<util:RemoveFolderEx>` action below. My guess would be that we are already cleaning up after ourselves on uninstall, so I think you'd be good to go on windows. Again, let's have @derekwbrown comment on this. Also, this should be very easy to test during QA.